### PR TITLE
12.0 AFC-1330 hr_employee_ciam_client

### PR DESCRIPTION
**Pull request by SebastianGieser-Vertel**

### CODE REVIEW

**Comment by FredrikArvas** [comment]
I believe the system-id should be AFDAFA in test.
image

File: hr_employee_ciam_client/models/client_config.py [hunk 1]
```diff
@@ -54,18 +54,18 @@ class ClientConfig(models.Model):
 
     def request_call(self, method, url, payload=False,
                      headers=False, params=False):
-
+
```
Comment (arturchilangwa-vertel) [inline hr_employee_ciam_client/models/client_config.py]: Unnecessary space

**Review by arturchilangwa-vertel — CHANGES_REQUESTED** [review]
Only one small change needed!

**Review by arturchilangwa-vertel — APPROVED** [review]

File: hr_employee_ciam_client/models/client_config.py [hunk 2]
```diff
@@ -87,10 +86,10 @@ def create_request_history(self, method, url, response, payload=False,
     def get_headers(self):
         tracking_id = pycompat.text_type(uuid.uuid1())
         headers = {
-            'x-amf-mediaType': "application/json",
+            'Content-Type': "application/json",
             'AF-TrackingId': tracking_id,
-            'AF-SystemId': "AF-SystemId",
-            'AF-EndUserId': "AF-EndUserId",
+            'AF-SystemId': "CRM",
+            'AF-EndUserId': "*sys*",
```
Comment (arturchilangwa-vertel) [inline hr_employee_ciam_client/models/client_config.py]: Check Fredriks comment on this
https://github.com/vertelab/odoo-hr/pull/34#issuecomment-744648576

**Review by arturchilangwa-vertel — CHANGES_REQUESTED** [review]
https://github.com/vertelab/odoo-hr/pull/34#issuecomment-744648576

**Review by arturchilangwa-vertel — APPROVED** [review]

**Pull request merged**